lib/internal/process/per_thread.js

```diff
@@ -46,7 +46,7 @@ const {
   validateNumber,
   validateObject,
 } = require('internal/validators');
-const { getValidatedPath } = require('internal/fs/utils');
+let getValidatedPath;
 const constants = internalBinding('constants').os.signals;
```

Comment:
```suggestion
const constants = internalBinding('constants').os.signals;

let getValidatedPath; // We need to lazy load it because of the circular dependency.
```